script.module.buggalo/resources/skins/Default/720p/buggalo-dialog.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As far as I remember, this skin file is pretty much designed around confluence. Have you checked how it looks in the default skin. Maybe try to provide a new, more up to date, default layout for the module

Reply:
I tested with the default skin "estuary" and it still looks ok, similar to the first version:
http://forum.kodi.tv/showthread.php?tid=121925&pid=1137307#pid1137307
From my point of view, it looks good enough. If more addons use this, someone with knowledge in skin design could have a look at this.

Reply:
Sure